ttfautohint-build.sh

```diff
@@ -42,7 +42,7 @@ TTFAUTOHINT_BIN="$INST/bin/ttfautohint"
 
 # The library versions.
 FREETYPE_VERSION="2.9"
-HARFBUZZ_VERSION="1.7.5"
+HARFBUZZ_VERSION="1.7.6"
```

Comment:
updates the Harfbuzz dependency to v1.7.6 

Harfbuzz changelog available in releases text:

https://github.com/harfbuzz/harfbuzz/releases